media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -948,6 +948,31 @@ textarea:focus {
 
     .container {
         padding-top: 0;
+
+        .tasks-more {
```

Comment:
With the <code>button-flat</code> class you won't need all these other properties, you might just need one for the font size.